base_layer/core/src/mempool/unconfirmed_pool/unconfirmed_pool.rs

```diff
@@ -292,8 +292,8 @@ mod test {
         let mut tx5 = tx!(MicroTari(5_000), fee:MicroTari(50), inputs:3, outputs:1).0;
         let mut tx6 = tx!(MicroTari(5_000), fee:MicroTari(75), inputs: 2, outputs: 1).0;
         // tx1 and tx5 have a shared input. Also, tx3 and tx6 have a shared input
-        tx5.body.inputs[0] = tx1.body.inputs[0].clone();
-        tx6.body.inputs[1] = tx3.body.inputs[1].clone();
+        tx5.body.inputs_mut()[0] = tx1.body.inputs()[0].clone();
+        tx6.body.inputs_mut()[1] = tx3.body.inputs()[1].clone();
```

Comment:
(1) This is the only place `inputs_mut` is used; it'd be great not to have to have this fn

Reply:
We could use `#[cfg(test)]` to only make the function available for testing.

Reply:
That's what we're doing right now.